### PR TITLE
Bugfix/documentation typo fixes

### PR DESCRIPTION
**Pull request by dosentmatter**
## Description

Typo fixes

## Checklist

* [x] I have read [CONTRIBUTING.md](../CONTRIBUTING.md)
* [x] I have linked all related issues above
* [x] I have rebased my branch

For bugfixes:
* [N/A] I have added at least one unit test to confirm the bug have been fixed
* [x] I have checked and updated TOC and API Docs when necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -830,15 +830,15 @@ gross(usd); // ok
...
-[⇧ back to top](#flows-utility-types)
+[⇧ back to top](#table-of-contents)
 
 ---
 
 ## Flow's Utility Types
 
 ### `$Keys<T>`
 
-get the union type of all the keys in an object type `T`
```
Comment (piotrwitek) [inline README.md]: I would like to get back these trailing whitespaces, because of a nice line-break to put link on a separate lines.

Comment (dosentmatter) [reply README.md]: What do you think about using inline HTML `<br>` tag instead? I think it's a bit more readable and searchable. Also, if we go and do a find&replace of trailing whitespace (or add it to the linter/prettier/editorconfig), we don't have to worry about messing up the double trailing whitespace.

If you prefer the double trailing whitespace in markdown. I can revert it.

Comment (piotrwitek) [reply README.md]: Great idea, I love it! 👍

Comment (dosentmatter) [reply README.md]: I added back the line breaks with `<br>`.

Also added alias for `Writable<T>`. Forgot to do it in #27.

Comment (piotrwitek) [reply README.md]: Alias looks good, although it's not re-exported from index.ts for public access.

Comment (dosentmatter) [reply README.md]: Whoops, my bad. I re-exported it in latest commit now.

**Comment by piotrwitek** [comment]
Looks great, thanks for the contribution! 🎉

**Pull request merged**